core/src/main/scala/utils/RichImplicits.scala

```diff
@@ -33,4 +34,9 @@ object Implicits {
     }
   }
 
+  implicit class RichChart(val chart: ChartAtom) extends AnyVal {
+    def newDefaultHtml(html: String): String =
+      html.replace("<script>", "<gu-script>").replace("</script>", "</gu-script>")
```

Comment:
Nice! Though kinda gross!

Worth having a comment here to explain why we do this?

Reply:
Yeah, will do in next update, although I'm hoping to generate all the SVG in the twirl template so there is no iframe anymore.